tests/src/kakarot/accounts/test_account_contract.py

```diff
@@ -351,176 +348,508 @@ def test_should_execute_starknet_call(self, cairo_run):
                 calldata=calldata,
             )
 
-    class TestValidate:
-        @pytest.mark.parametrize("seed", (41, 42))
-        @pytest.mark.parametrize("transaction", TRANSACTIONS)
+    class TestExecuteFromOutside:
         @SyscallHandler.patch(
-            "Account_cairo1_helpers_class_hash", CAIRO1_HELPERS_CLASS_HASH
+            "IKakarot.get_cairo1_helpers_class_hash",
+            lambda addr, data: [CAIRO1_HELPERS_CLASS_HASH],
         )
```

Comment:
this test doesn't require this patch

Reply:
and actually this would be better set directly in the syscall_handler.py, like `ACCOUNT_CLASS_IMPLEMENTATION`